packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -176,7 +176,14 @@ export type EVMContext<CN extends string> = {
   lca: LocalAccount;
   gmpFee: DenomAmount;
   gmpChain: Chain<{ chainId: string }>;
-  addresses: Record<CN | 'usdc', EVMT['address']>;
+  addresses: Record<
+    | CN
+    | 'usdc'
+    | 'aaveRewardsController'
+    | 'aaveUSDC'
+    | 'compoundRewardsController',
```

Comment:
Only `usdc` is common to all `EVMContext`s. These belong on the individual `ProtocolDetail`s.
```suggestion
```